From broonie@opensource.wolfsonmicro.com  Wed Jan 27 14:36:19 2010
From: Mark Brown <broonie@opensource.wolfsonmicro.com>
Date: Mon, 18 Jan 2010 12:03:18 +0000
Subject: USB: r8a66597-udc: Prototype IS_ERR() and PTR_ERR()
To: David Brownell <dbrownell@users.sourceforge.net>, Paul Mundt <lethal@linux-sh.org>, Magnus Damm <damm@igel.co.jp>, Yoshihiro Shimoda <shimoda.yoshihiro@renesas.com>
Cc: linux-usb@vger.kernel.org, Mark Brown <broonie@opensource.wolfsonmicro.com>
Message-ID: <1263816198-32008-1-git-send-email-broonie@opensource.wolfsonmicro.com>


The build of r8a66597-udc was failing on ARM since IS_ERR() and
PTR_ERR() weren't protyped.  Presumably err.h is being pulled in by
another header on other platforms.

Signed-off-by: Mark Brown <broonie@opensource.wolfsonmicro.com>
Acked-by: Yoshihiro Shimoda <shimoda.yoshihiro@renesas.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/gadget/r8a66597-udc.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/gadget/r8a66597-udc.c
+++ b/drivers/usb/gadget/r8a66597-udc.c
@@ -26,6 +26,7 @@
 #include <linux/io.h>
 #include <linux/platform_device.h>
 #include <linux/clk.h>
+#include <linux/err.h>
 
 #include <linux/usb/ch9.h>
 #include <linux/usb/gadget.h>
